src/main/java/com/opencastsoftware/prettier4j/Doc.java

```diff
@@ -849,6 +858,67 @@ public String toString() {
         }
     }
 
+    /**
+     * Represents an aligned {@link Doc}.
+     *
+     * Sets the indentation for line breaks within its inner {@link Doc} at the current line position.
+     */
+    public static class Align extends Doc {
```

Comment:
It may also be worth adding an absolute `Position` variant of `Align` that takes a user-provided `position` argument to be applied after line breaks